cookbooks/kinematically_driven_subduction_2d/kinematically_driven_subduction_2d_case2b.prm

```diff
@@ -0,0 +1,46 @@
+# This cookbook models oceanic subduction in a 2d
+# box of 3000 x 670 km.
+# The subducting plate is driven by inflow boundary
+# conditions on the top part of the right boundary,
+# while a balancing outflow is prescribed underneath.
+# Only density and viscosity differ per composition.
+# Building on case 2a, an initial temperature field
+# based on the plate cooling model and
+# a mantle adiabatic gradient of 0.25 degrees/km
+# is prescribed. Compared to case2a, this model includes
+# a temperature dependent density.
+# High resolution is required to a.o. resolve the weak zone
+# between the subducting and overriding plate, so about
+# 100 MPI processes are required.
+# We include a shared library for the initial temperature plugin and the isotherm depth, composition RMS temperature, and trench location postprocessors.
+set Additional shared libraries            = $ASPECT_SOURCE_DIR/cookbooks/kinematically_driven_subduction_2d/libsubduction_plate_cooling.so
+
+# 2D
+set Dimension                              = 2
+
+# Reuse parts of case 2a
+include kinematically_driven_subduction_2d_case2a.prm
+
+set Output directory                       = output-Case2b
+
+subsection Material model
+  set Model name = multicomponent
+
+  subsection Multicomponent
```

Comment:
If any of the parameters below are reused from case 2a (Thermal conductivities, Heat Capacities), consider removing them. I would always add spaces between different sets of parameters to improve the readability.